src/client/app/components/search/resultDetail.js

```diff
@@ -0,0 +1,17 @@
+import React, { PropTypes } from 'react';
+
```

Comment:
I don't think this will pass the linting process, cf.
```
const ResultDetail = ({ artist, album, releaseDate }) => (
    <div>
        <p>by&nbsp;{artist}</p>
        <p>on&nbsp;{album}&nbsp;released {releaseDate}</p>
    </div>
);
```
It would also be great to find a CSS solution to bolding the text, whether that be wrapping the expressions in span tags or otherwise, since `b` is technically not used for styling text anymore: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b

Reply:
Updated. Ran linter this time!

Reckon spans with inline styles an ok replacement for `<b>`s? Feels gross but I guess so are `<b>`s. Normally I'd add classes and style them in the scss but this feels more reacty and idk if that makes it more correct.